### PR TITLE
Fix Possible NullPointerException

### PR DESCRIPTION
**Pull request by DoctaEnkoda**
Fix Possible NullPointerException With Essentials

### CODE REVIEW

**Review by dlsf — APPROVED** [review]

File: src/main/java/com/iridium/iridiumskyblock/utils/PlayerUtils.java [hunk 1]
```diff
@@ -95,10 +95,12 @@ public static void teleportSpawn(Player player) {
         if (Bukkit.getPluginManager().isPluginEnabled("EssentialsSpawn")) {
```
Comment (PeachesMLG) [inline src/main/java/com/iridium/iridiumskyblock/utils/PlayerUtils.java:95]: Dont we already check for that here?

Comment (DoctaEnkoda) [reply src/main/java/com/iridium/iridiumskyblock/utils/PlayerUtils.java:95]: API says the answer can be null, better avoid a problem in the future

Comment (DoctaEnkoda) [reply src/main/java/com/iridium/iridiumskyblock/utils/PlayerUtils.java:95]: Crash Plugin, Disabled with an issue...

Comment (PeachesMLG) [reply src/main/java/com/iridium/iridiumskyblock/utils/PlayerUtils.java:95]: ok, well we should remove the if statement above and replace it with your ones then, seems kinda dum to have both?

**Review by PeachesMLG — CHANGES_REQUESTED** [review]
Discussed changes

**Comment by PeachesMLG** [comment]
Fixed Issue with if statements

**Pull request merged**